src/test/java/camp/nextstep/edu/rebellion/SetCollectionTest.java

```diff
@@ -0,0 +1,56 @@
+package camp.nextstep.edu.rebellion;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 size가 정상적으로 크기를 반환하는지 확인")
+    @Test
+    public void sizeTest() {
+        // when & then
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
다음과 같이 테스트 할 수 있을것 같네요 :)
```suggestion
        assertThat(numbers).hasSize(3);
```